gusto/timestepping/semi_implicit_quasi_newton.py

```diff
@@ -368,9 +393,20 @@ def timestep(self):
                 self.io.log_courant(self.fields, 'transporting_velocity',
                                     message=f'transporting velocity, outer iteration {outer}')
                 for name, scheme in self.active_transport:
-                    logger.info(f'Semi-implicit Quasi Newton: Transport {outer}: {name}')
-                    # transports a field from xstar and puts result in xp
-                    self.transport_field(name, scheme, xstar, xp)
+                    if isinstance(name, list):
```

Comment:
Can we find a way to move this if statement and the comments into the `transport_field` routine? I would quite like to keep this part of the timeloop looking as simple as:
`self.transport_field(...)`

For instance maybe we should move the iteration into `transport_field` so this call becomes:
```
self.transport_field(xstar, xp)
```

and `self.transport_field` is defined so that:
```
def transport_field(self, xstar, xp):
    for name, scheme in self.active_transport:
        ... # do everything we previously did
```

Reply:
I agree that this is a nice way of doing it. I've implemented this change, which assumes that we won't want to use a divergence predictor with simultaneous transport.